packages/components/src/popover/types.ts

```diff
@@ -138,7 +134,17 @@ export type PopoverProps = {
 	 * @default false
 	 */
 	shift?: boolean;
-
+	/**
+	 * Specifies the popover's style.
+	 *
+	 * Leave undefined for the default style. Other values are:
+	 * - 'unstyled':  The popover is essentially without any visible style, it
+	 *                has no background, border, outline or drop shadow, but
+	 *                the popover contents are still displayed.
+	 * - 'alternate': A style that has no elevation, but a high contrast with
+	 *                other elements.
```

Comment:
Would this be a good opportunity to give the "alternate" variant a more meaningful name? Even something like `highContrast` as you put it in this description feels better to me. Would that be an appropriate encapsulation of the design intent? (cc @jasmussen)

Reply:
That's a good shout, I wondered that too.

Reply:
Thanks for the ping. I don't have a strong opinion on the name, though I don't think we should call it high-contrast. In fact ideally the variant name can become entirely hidden and automatic. The thing is: when a popover opens from the block toolbar, the popover should be the same visual style as the block toolbar (with the dark border). If it doesn't open from a block toolbar, it should be the default shadowed light gray material. Could this be made automatic so as a dev, one doesn't have to choose this variant?

Reply:
Thanks for the explanation, that helps a lot.

Then we could perhaps call the variant `"toolbar"`, in association with the [Toolbar](https://wordpress.github.io/gutenberg/?path=/docs/components-toolbar--default) component. @talldan @ciampo Thoughts?

> Could this be made automatic so as a dev, one doesn't have to choose this variant?

Yes, I think we could. Moved to an issue: #45246

Reply:
`'toolbar'` sounds like a good idea 👍

Reply:
I've updated the PR to use that naming.

Reply:
👋 from the future. We now have top toolbar which has different border settings. The floating toolbar popover has a higher contrast black border and the top toolbar has the default shadowed gray border. They are both opened from the toolbar, so the name `toolbar` no longer communicates the difference. I think changing it to something that indicates the style like `highContrast`, etc makes more sense now.

I've looked into this a little bit and it seems there's a lot to unravel here, so I'm sure there's some context and considerations I'm missing. Given the current state of the toolbars, what do y'all think?

Reply:
Naming things is hard — the `toolbar` naming was in place to suggest that the popover would look exactly like the toolbar — but then the toolbar also [got a style variant](https://github.com/WordPress/gutenberg/pull/55139).

`highContrast` sounds like a more descriptive name, although that means that, should the styles required by the toolbar change to something that doesn't look very "high contrast", we will probably have to introduce a new variant, and keep `highContrast` around.

If were to implement those changes, we would have to deprecate the `toolbar` variant name, keep it around for backwards compat, and refactot the toolbar, dropdown and dropdownmenu components to use the new `highContrast` variant instead.